ISISScientificComputing/autoreduce
# Data archive explorer

# Changes
The archive explorer is designed to be able to provide an easy way to obtain data from the isis data archive.
this allows us to remove a lot of the functionality and cached values that were previously stored in the data archive creator. This means that we can now use this searching functionality with both a real and a fake archive

# To test
* Ensure that the new unit tests for the archive explorer and file filter run and pass.
* Ensure the new files have 100% test coverage in coveralls
* [Creating a fake archive](https://github.com/ISISScientificComputing/autoreduce/wiki/Testing-and-validating#creating-a-fake-data-archive) locally. Create an `AchiveExplorer` and check it find all the value you expect. 

Fixes #139
